packages/services/queue-pg-boss/run-tests.sh

```
@@ -1,3 +1,5 @@
 #!/bin/bash
 cd "$(dirname "$0")"
-node --loader tsx --test --experimental-test-coverage src/**/*.test.ts "$@"
\ No newline at end of file
+files=($(find src -type f -name "*.test.ts"))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)